lib/runner.js

```diff
@@ -46,6 +46,10 @@ var Runner = function(config) {
     flow.timeout(1000, 'waiting for debugger to attach');
   }
 
+  if (config.debuggerServerPort) {
+    process.debuggerServerPort = config.debuggerServerPort;
```

Comment:
Protractor doesn't have access to the config object. Alternatively, we can create a setter in Protractor and set it in runner or something, but this seems very roundabout for a non-production feature.